broccoli/features.js

```diff
@@ -1,35 +1,12 @@
 'use strict';
 
-const fs = require('fs');
-const ts = require('./typescript');
-
+const requireEsm = require('esm')(module);
 function getFeatures() {
-  let fileName = 'packages/@ember/canary-features/index.ts';
-  let fileContents = fs.readFileSync(fileName).toString();
-
-  let sourceFile = ts.createSourceFile(
-    fileName,
-    fileContents,
-    ts.ScriptTarget.ES2017,
-    /*setParentNodes */ true
+  const { default: features } = requireEsm(
+    '../packages/@ember/canary-features/default-features.js'
```

Comment:
I could not use `@ember/canary-features/default-features.js` for the path. Would this be because ember.js' internal package structure differs from ember data?

Reply:
I think the local require (e.g. starting with `../packages/`) is totally correct here.